wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md

```diff
@@ -26,7 +26,27 @@ Follow these steps to set up a local playground for the theme:
     4.  `cd` back to the Openverse theme directory at `../wporg-openverse`
         (i.e. `wordpress.org/wordpress.org/public_html/wp-content/themes/pub/wporg-openverse`).
 
-2.  You can choose to set up a new environment automatically or work in an
+2.  Build the MU plugins.
+
+    1.  Clone the `WordPress/wporg-mu-plugins` repo right into this directory.
+        It's `.gitignored` so it shouldn't affect anything.
```

Comment:
Can you clarify exactly which directory to do this in?

Also, I can't find any gitignore files in this repository at all...

Reply:
This is the `.gitignore` I'm referring to
```
wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/.gitignore
```